controller_manager:
  ros__parameters:
    update_rate: 1 # Hz

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster


forward_position_controller:
  ros__parameters:
    type: forward_command_controller/ForwardCommandController
    joints:
      - joint2
      - joint5
    interface_name: position
